### DISCUSSION TITLE
Grove - Finger-clip Heart Rate Sensor | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1154] [Q&A]
# Grove - Finger-clip Heart Rate Sensor | Seeed Studio Wiki

Grove - Finger-clip Heart Rate Sensor

https://wiki.seeedstudio.com/Grove-Finger-clip_Heart_Rate_Sensor/

**Comment by 9-Fakrizz** [comment]
hi, I bought this one and I followed up tutorial but it's not work . the wire.request never sent back any values.
so, i try to check addr with scanner then i found that it's 0x50 instead of 0xA0 (from code tutorial). So, i try wire.request again with new addr that i just found. the result was the same it never sent back any values.
All of these, i could say my wiring are ok (because esp32 found addr of grove) and also grove no any short circuit(using multimeter). So how can i fix this ?
